repos/base/src/ld/genode.ld

```diff
@@ -15,7 +15,7 @@ ENTRY(_start)
 
 PHDRS
 {
-	ro PT_LOAD;
+	ro PT_LOAD FLAGS(5);
```

Comment:
I like this clear specification. Would you mind to also add `FLAGS` for the `rw` segment?

Reply:
Sure, see the updated version of the patch.